dashboard/src/views/Projects/Versions/index.tsx

```diff
@@ -56,18 +51,22 @@ export const VersionsOverview = () => {
           "& > *": {
             minWidth: { xs: "120px", md: "160px" },
           },
+          flexDirection: 'column',
         }}
       >
         <Typography
           level="h2"
           component="h1"
-          style={{
+          sx={{
             alignSelf: "center",
             fontSize: "3rem",
             marginRight: "1.5rem",
+            backgroundColor: '#0078ff',
+            p: '1rem 5rem',
+            borderRadius: "sm",
           }}
         >
-          Versions
+          {project?.name} versions
```

Comment:
is versions always plural here ;) ? a little gotcha

Reply:
I think "FlowUI versions" give more context on what is that page about. Something like "FlowUI version" sounds a bit weird and can sound confusing